manager/api/rest-impl/src/main/java/io/apiman/manager/api/rest/impl/ActionResourceImpl.java

```diff
@@ -89,17 +77,19 @@
 
     @Inject @ApimanLogger(ActionResourceImpl.class) IApimanLogger log;
 
+    private OrganizationResourceImpl organizationResource;
+
```

Comment:
Is this used?

Reply:
No, I don't think so. Well paid attention 👍 

I will check this again and provide a fixup soon :)